generate-default.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
[dust] Gotta support those NixOS users that only have `/bin/sh` and `/usr/bin/env` 😉 
```suggestion
#!/usr/bin/env bash
```